xskillscore/core/deterministic.py

```diff
@@ -205,6 +207,55 @@ def pearson_r(a, b, dim, weights=None, skipna=False):
     )
 
 
+def r2(a, b, dim, weights=None, skipna=False):
```

Comment:
Kind of strange to have a two letter function; maybe r_squared?

Reply:
I am in favor of short names. e.g. `mape`, `rmse`. People who use this metric will be familiar with `r2_score` from sklearn